samples/snippets/requirements.txt

```
@@ -1,33 +1,33 @@
-alembic==1.13.1
-certifi==2024.7.4
-charset-normalizer==3.3.2
-geoalchemy2==0.15.1
-google-api-core[grpc]==2.19.0
-google-auth==2.29.0
-google-cloud-bigquery==3.24.0
+alembic==1.14.0
+certifi==2024.8.30
+charset-normalizer==3.4.0
+geoalchemy2==0.16.0
+google-api-core[grpc]==2.23.0
+google-auth==2.36.0
+google-cloud-bigquery==3.27.0
 google-cloud-core==2.4.1
-google-crc32c==1.5.0
-google-resumable-media==2.7.0
-googleapis-common-protos==1.63.1
-greenlet==3.0.3
-grpcio==1.62.2
-grpcio-status==1.62.2
-idna==3.7
-importlib-resources==6.4.0; python_version >= '3.8'
-mako==1.3.5
-markupsafe==2.1.5
-packaging==24.0
-proto-plus==1.23.0
+google-crc32c==1.6.0
+google-resumable-media==2.7.2
+googleapis-common-protos==1.66.0
+greenlet==3.1.1
+grpcio==1.67.1
+grpcio-status==1.67.1
+idna==3.10
+importlib-resources==6.4.5; python_version >= '3.8'
+mako==1.3.6
+markupsafe==3.0.2
+packaging==24.2
+proto-plus==1.25.0
 protobuf===4.25.3
```

Protobuf should be updated automatically : 5.28.3

```suggestion
protobuf==5.28.3
```